src/main/java/move/GeneralMovement.java

```diff
@@ -0,0 +1,29 @@
+package move;
+
+import direction.Point;
+import piece.Pieces;
+
+public class GeneralMovement implements MovementRule {
+
+    public static final int MAXIMUM_MOVE_RANGE = 1;
+
+    @Override
+    public Point move(Pieces pieces, Point from, Point to) {
+        validatePossibleHorizontalMovement(from, to);
+        validatePossibleVerticalMovement(from, to);
+
+        return to;
+    }
+
+    private void validatePossibleHorizontalMovement(Point from, Point to) {
+        if (from.x() + MAXIMUM_MOVE_RANGE < to.x() || from.x() - MAXIMUM_MOVE_RANGE > to.x()) {
```

Comment:
```suggestion
        if ((from.x() + MAXIMUM_MOVE_RANGE) < to.x() || (from.x() - MAXIMUM_MOVE_RANGE) > to.x()) {
```
우선순위가 헷갈리지 않도록 작성해보아요.

Reply:
생각못한 부분이네요. 해당 피드백 반영 하겠습니다!

커밋: [200cba3](https://github.com/woowacourse/java-janggi/pull/72/commits/200cba31ac7d0db7175f4e5c6c592fc7b0c69b89)